pkg/flow/adapter/xslttransformation/adapter_test.go

```diff
@@ -171,21 +220,6 @@ func TestXSLTTransformationEvents(t *testing.T) {
 			expectEvent:    newCloudEvent(tAlternativeOutXML, cloudevents.ApplicationXML),
 			expectCategory: tSuccessAttribute,
 		},
-
-		"wrong XSLT": {
-			allowXSLTOverride: false,
-			xslt:              tFaultyXML,
-
-			expectPanic: "XSLT validation error: failed to parse xsl",
-		},
-
-		"wrong configuration not providing XSLT": {
-			allowXSLTOverride: false,
-			xslt:              "",
-
-			expectPanic: "if XSLT cannot be overriden by CloudEvent payloads, configured XSLT cannot be empty",
-		},
```

Comment:
I relocated these two test cases to a new `TestNewTarget` test, because they were only testing for panic in the constructor, instead of performing assertions on events.

I think it's more clear that way because we test the constructor and the dispatcher separately (one doesn't influence the other in the production code).
Comments welcome.